pkg/flagset/storagehome.go

```diff
@@ -329,10 +329,20 @@ func StorageHomeWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "storage-owncloud-layout",
-			Value:       "{{.Username}}",
-			Usage:       `"layout of the users home dir path on disk, in addition to {{.Username}}, {{.UsernameLower}} and {{.Provider}} also supports prefixing dirs: "{{.UsernamePrefixCount.2}}/{{.UsernameLower}}" will turn "Einstein" into "Ei/Einstein" `,
+			Value:       "{{.Id.OpaqueId}}",
+			Usage:       `"layout of the users home dir path on disk, in addition to {{.Username}}, {{.Mail}}, {{.Id.OpaqueId}}, {{.Id.Idp}} also supports prefixing dirs: "{{substr 0 1 .Username}}/{{.Username}}" will turn "Einstein" into "Ei/Einstein" `,
 			EnvVars:     []string{"REVA_STORAGE_OWNCLOUD_LAYOUT"},
 			Destination: &cfg.Reva.Storages.OwnCloud.Layout,
 		},
+
+		// User provider
+
+		&cli.StringFlag{
+			Name:        "users-url",
+			Value:       "localhost:9144",
```

Comment:
I remember people complaining here, that a URL should always have `http(s)://`. Please adjust. ;-)